guide/how-it-works/silent-payments.md

```diff
@@ -0,0 +1,360 @@
+      modalImage: setup-silent-payments-onboarding-iii
+      alt: .
+      caption: .
+
+images_send:
+    - file: send-review-sp
+      modalImage: send-review-sp
+      alt: .
+      caption: .
+    - file: send-review-with-contact
+      modalImage: send-review-with-contact
+      alt: .
+      caption: .
+    - file: send-address-error
+      modalImage: send-address-error
+      alt: .
+      caption: .
+
+images_test-txns:
+    - file: test-txn-send-review
+      modalImage: test-txn-send-review
+      alt: .
+      caption: .
+    - file: test-txn-send-home
+      modalImage: test-txn-send-home
+      alt: .
+      caption: .
+    - file: test-txn-confirm-finalise-txn
+      modalImage: test-txn-confirm-finalise-txn
+      alt: .
+      caption: .
+
+images_backup:
+    - file: backup-cloud-backup
+      modalImage: backup-cloud-backup
+      alt: .
+      caption: .
+    - file: backup-wallet-birthday
+      modalImage: backup-wallet-birthday
+      alt: .
+      caption: .
+
+images_recovery:
+    - file: recovery-wallet-birthday-prompt
+      modalImage: recovery-wallet-birthday-prompt
+      alt: .
+      caption: .
+    - file: recovery-wallet-birthday-enter
+      modalImage: recovery-wallet-birthday-enter
+      alt: .
+      caption: .
+---
+
+<!--
+
+Editor's notes
+
+This page provides best practices and designers for the first-use experience of the
+daily spending wallet reference design.
+
+Source designs for all screens can be found in this Figma file:
+https://www.figma.com/file/lf2Xyw2I2OXPsHiFQVQdiG/Daily-spending-wallet-prototype?node-id=4152%3A306968&t=boEAgdyqfi39SBoF-1
+
+Source file for header image:
+https://www.figma.com/file/NjtMNQiJtoVkedEHgwD0A9/BTC-DSN-Guide-Header-Images?node-id=0%3A1&t=JzVvsbPdFpjhNTGF-1
+
+-->
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/silent-payments/silent-payments-header.png"
+   retina = "/assets/images/guide/how-it-works/silent-payments/silent-payments-header@2x.png"
+   mobile = "/assets/images/guide/how-it-works/silent-payments/silent-payments-header-mobile.png"
+   mobileRetina = "/assets/images/guide/daily-spending-wallet/silent-payments/silent-payments-header-mobile@2x.png"
+   alt-text = "."
+   width = 1600
+   height = 1000
+   layout = "full-width"
+%}
+
+# Silent payments
+{: .no_toc }
+
+---
+
+<div class="glossary-toc" markdown="1">
+ * Table of contents
+{:toc}
+</div>
+
+---
+Bitcoin addresses are only meant to be used once. Thus, transaction flows usually start with users interacting with each other to specify the on-chain address(es) to be used in the transaction, with the goal of preventing on-chain address reuse. This not only adds steps to the user flow and adds friction by way of time and mental overhead on the part of users. BIP-352 specifies a protocol called Silent Payments that allows users to avoid such an interaction altogether, while ensuring on-chain addresses are not reused through a new type of static payment addresses.
```

Comment:
Good to link to the [address page](https://bitcoin.design/guide/glossary/address/) in the glossary, and also the BIP itself.

Wonder if it isn't better to start this intro paragraph with something like "Silent payments is a protocol and address type that improves/simplifies...", instead of the line about single use of addresses?

Reply:
Rephrased the entire intro and add reference to a nice section about address reuse in https://github.com/BitcoinDesign/Guide/pull/1109/commits/793aa9f23fb4eff0a76f0a41c5a817d60ab271c6